src/test/java/io/github/externschool/planner/config/SpringSecurityConfigTest.java

```diff
@@ -22,7 +22,7 @@
 
     @Test
     public void userIsAuthenticatedTest() throws Exception {
-        mockMvc.perform(formLogin().user("admin@").password("admin"))
+        mockMvc.perform(formLogin().user("admin@a").password("admin"))
```

Comment:
Also we can check redirection on `/` after successful auth

Reply:
that redirection to "/" is made by framework without any help of mine. i doubt i'm better than Juergen Hoeller in spring testing

Reply:
Yes, but if someone changes this way by default, this test will not work